src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx

```diff
@@ -76,18 +76,20 @@ Software built at New Relic goes through these five phases.
+Each project team for our generally available New Relic One platform is assigned a security engineer charged with reviewing and advision on the security of New Relic products. In the requirements building phase the security engineer performs a risk assessment and then adds security requirements for the project. Privacy and compliance experts are added to the project teams as needed.
 
 ## Design
-During the design phase, the New Relic Security team collaborates with the stakeholders, engineering leaders, and architects to get a detailed shared understanding of the feature. Security engineers at New Relic contribute to the design process with stakeholders by creating a threat model documenting any acceptance criteria, features, or requirements to securely implement the feature. Using the threat model, the security engineer adds detailed specifications for the required controls to the project.
+During the design phase, the New Relic security engineer collaborates with the stakeholders, engineering leaders, and architects to get a detailed shared understanding of the feature. Security engineers at New Relic contribute to the design process with stakeholders by creating a threat model documenting any acceptance criteria, features, or requirements to securely implement the feature. Using the threat model, the security engineer adds  detailed specifications for the required controls to the project.
 
 ## Build
-In the build phase, the engineering team implements the features in the project following secure coding standards at New Relic. Each product engineer receives secure coding training, which includes topics such as the [OWASP top 10](https://owasp.org/www-project-top-ten/), input sanitization, and using the secure frameworks and processes already in place at New Relic.
+Each product engineer receives secure coding training which includes topics such as the [OWASP top 10](https://owasp.org/www-project-top-ten/), input sanitization, and using the secure frameworks and process already in place at New Relic. In the build phase the engineering team implements appropriate security features in the project following secure coding standards at New Relic.
 
 ## Verification
-Once feature complete, every pull request must be code reviewed by another engineer with write access to the repository. At the project level, teams may request a final review of the project before deploying. The security engineer verifies that the safeguards and controls discovered and documented in the risk assessment, requirements, and design phases have been addressed.
+Once code complete, every pull request must be code reviewed by another engineer with write access to the code repository. At the project level teams may request a final review of the project before deploying. The security engineer verifies the safeguards and controls discovered and documented in the risk assessment, requirements, and design phase have been addressed.
```

Comment:
I would rearrange that last sentence like this: The security engineer verifies the following have been addressed: safeguards and controls discovered and documented in the risk assessment, requirements, and design phases.

If this isn't accurate, let me know.

Reply:
Updated wording here with additional input from legal.